### PR TITLE
Move global static variables out of headers

### PR DESCRIPTION
**Pull request by GravisZro**
## Pull Request Type
- [x] Build and Dependency changes

### Description
A few headers had some global static variables declared in their headers which is very bad. I also moved a function containing a static because it's only referenced in the implementation file.

### Checklist
- [x] I have tested my changes locally and verified that they work as intended.
- [x] I have documented any new or modified functionality.
- [x] I have reviewed the changes to ensure they do not introduce any unnecessary complexity or duplicate code.
- [x] I understand that by submitting this pull request, I am agreeing to license my contributions under the project's license.

### CODE REVIEW

File: ddebug/debug.h [hunk 1]
```diff
@@ -134,11 +134,11 @@
...
-static const int OSMBOX_YESNOCANCEL = 3;
-static const int OSMBOX_ABORTRETRYIGNORE = 4;
-static const int OSMBOX_OKCANCEL = 5;
+constexpr const int OSMBOX_OK = 1;
+constexpr const int OSMBOX_YESNO = 2;
+constexpr const int OSMBOX_YESNOCANCEL = 3;
+constexpr const int OSMBOX_ABORTRETRYIGNORE = 4;
+constexpr const int OSMBOX_OKCANCEL = 5;
 #else
 #define OSMBOX_OK 1
```
Comment (Lgt2x) [inline ddebug/debug.h:143]: should these definitions go away?

**Review by Lgt2x — APPROVED** [review]

File: ddebug/debug.h (hunk 1 above)
Comment (GravisZro) [reply ddebug/debug.h:143]: That is not in the scope of this PR.

Comment (Lgt2x) [reply ddebug/debug.h:143]: I know, it was more of a general remark and an idea for a future cleanup

Comment (GravisZro) [reply ddebug/debug.h:143]: Oh, I see, the definition below. Yeah, they should go and the `constexpr` ones should be universal. Also there seems to be a duplicate sets that merely leave of "OS" from the beginning. I'll clean it up. Need to merge this first, though.

**Review by bryanperris — APPROVED** [review]

**Comment by GravisZro** [comment]
Let the merging begin?

**Pull request merged**